testing/scripts/have-javascript

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if grep -q "ZEEK_HAVE_JAVASCRIPT:INTERNAL=no" "${BUILD}"/CMakeCache.txt; then
```

Comment:
I've noticed a small hiccup with this — with libnode available, but configuring with `--disable-javascript`, CMakeCache.txt contains this:
```
//Zeek has JavaScript support
ZEEK_HAVE_JAVASCRIPT:INTERNAL=
```
That is, the grep doesn't catch and so the javascript tests don't get skipped but fail. (When libnode isn't actually on my system, it says `=no` as expected.)

Simply turn the test around to look specifically for `=yes` and I think all is well.

Reply:
Thanks for catching that.